Automattic/wp-calypso
# Marketplace: Eligibility warning proceed buttons

#### Proposed Changes

* Swaps button location to the right when loading the eligibility warnings in a dialog
* Removes dialog content padding and adds a border/shadow to the button area

Before

![Screenshot 2022-09-27 at 18-46-00 MailPoet – emails and newsletters in WordPress Plugin ‹ Business - Simple Site — WordPress com](https://user-images.githubusercontent.com/811776/192479422-64a23f31-cb28-4e4d-ac23-bc35fc30ca0a.png)

After

![Screenshot 2022-09-27 at 18-48-59 MailPoet – emails and newsletters in WordPress Plugin ‹ Business - Simple Site — WordPress com](https://user-images.githubusercontent.com/811776/192479635-f11cc3bb-6413-48c0-b2bf-a370d4777e9a.png)


#### Testing Instructions

* Confirm this applies to the CTAButton use case only
  * https://wordpress.com/plugins/mailpoet/testing705633960.wordpress.com (click install with a simple site)
* Other locations should be unaffected (all these require a business plan simple site otherwise you'll be taken to the checkout)
  * http://calypso.localhost:3000/themes/upload/test809856404.wordpress.com (click install theme on the theme page)
  * http://calypso.localhost:3000/backup/test809856404.wordpress.com (click activate)
  * http://calypso.localhost:3000/hosting-config/activate/test809856404.wordpress.com (hosting config > click activate)

Fixes #68248
